### PR TITLE
Starccm touchup

### PR DESCRIPTION
**Pull request by JonShelley**
Made some additional changes to support different versions and updated the run_case.pbs since it did not work as expected. Also updated the readme.md

### CODE REVIEW

**Comment by kanchanm** [comment]
@JonShelley there are some conflicts, can you merge with the latest?

File: apps/starccm/readme.md [hunk 1]
```diff
@@ -17,37 +17,43 @@ azhpc-scp -r $azhpc_dir/apps hpcuser@headnode:.
...
+azhpc-run -u hpcuser -n compute apps/starccm/scripts/add_reqs.sh 
 ```
 
 ### Install Starccm+
 
 You must first obtain the starccm installer and copy it to the cluster.  If it is available on the local machine you can copy as follows:
 
 ```
-azhpc-scp STAR-CCM+14.06.004_02_linux-x86_64-2.12_gnu7.1.tar.gz /mnt/resource/.
+azhpc-scp STAR-CCM+14.04.013_01_linux-x86_64-2.12_gnu7.1.zip /mnt/resource/.
```
Comment (kanchanm) [inline apps/starccm/readme.md]: this is already fixed in the latest checkin -- should be hpcuser@headnode:/mnt/resource/.

Comment (JonShelley) [reply apps/starccm/readme.md]: added the fix.

File: apps/starccm/run_case.pbs [hunk 2]
```diff
@@ -2,26 +2,29 @@
 
 # parameters that can be overridden
 APP_INSTALL_DIR=${APP_INSTALL_DIR:-/apps}
-DATA_DIR={DATA_DIR:-.}
-CASE={CASE:-civil}
-# PODKEY is required (pass in as environment variable)
-#PODKEY=
+DATA_DIR=${DATA_DIR:-/data/starccm}
+CASE=${CASE:-civil}
+# Valid OMPI values are openmpi and openmpi4
```
Comment (kanchanm) [inline apps/starccm/run_case.pbs]: hmm.. isn't it just openmpi at this time? would suggest not mentioning this.

Comment (JonShelley) [reply apps/starccm/run_case.pbs]: Addressed comment

**Pull request merged**